back/app/services/survey_results_generator_service.rb

```diff
@@ -124,18 +106,19 @@ def core_field_attributes(field, response_count)
     }
   end
 
-  def visit_select_base(field, group_mode: nil, group_field_id: nil)
+  def visit_select_base(field)
     query = inputs
-    if group_field_id
-      if group_mode == 'user_field'
+    if @group_field_id
+      if @group_mode == 'user_field'
```

Comment:
might be worth accessing these via attribute accessors as we do fields and inputs